react/react1/week2/todo-app/src/components/TodoList.js

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+import { todos } from '../data';
+import Button from './Button';
+import TodoRow from './TodoRow';
+
+const TodoList = () => {
+  const [todosList, setTodosList] = useState(todos);
+
+  const newTodo = {
+    id: todosList.length + 1,
+    description: `random text with id: ${todosList.length + 1}`,
+  };
+
+  function addTodo() {
+    setTodosList([...todosList, newTodo]);
+  }
+
+  function updateTodoList(id) {
+    const updetedList = todosList.filter((el) => el.id !== id);
+
+    setTodosList(updetedList);
+  }
+
+  return (
+    <>
+      <Button onClick={addTodo} bigSize text='Add todo' />
+      <div>
+        {todosList.map((item, index) => (
+          <TodoRow
+            key={index}
```

Comment:
Having `index` as the key, is not a good idea, since the they can be removed and be added and then React will loose it's advantage of using those unique Ids, so it's better to use your `item.id`